src/injections/css/bug1568256-zertifikate.commerzbank.de-flex.css

```diff
@@ -0,0 +1,12 @@
+/**
+ * zertifikate.commerzbank.de - clickable elements on the page are collapsed
+ * Bug #1568256 - https://bugzilla.mozilla.org/show_bug.cgi?id=1568256
+ * WebCompat issue #9102 - https://webcompat.com/issues/9102
+ *
+ * Affected elements have display:-webkit-box and display:flex applied, however,
+ * listed in wrong order, so display:-webkit-box is becoming the final say.
+ * Adding display: flex for those elements fixes the issue
+ */
+.x-layout-box {
+    display: flex!important;
```

Comment:
Nit: Two spaces instead of four, space before `!important`. (That should have also applied to the google DNS override, but I missed that one, so let's let that slide.)